go-controller/pkg/ovn/ovn.go

```diff
@@ -549,9 +561,62 @@ func (oc *Controller) recordPodEvent(addErr error, pod *kapi.Pod) {
 	}
 }
 
+// iterateRetryPods checks if any outstanding pods have been waiting for 60 seconds of last known failure
+// then tries to re-add them if so
+func (oc *Controller) iterateRetryPods() {
+	oc.retryPodsLock.Lock()
+	defer oc.retryPodsLock.Unlock()
+	now := time.Now()
+	for uid, podEntry := range oc.retryPods {
+		pod := podEntry.pod
+		podTimer := podEntry.timeStamp.Add(time.Minute)
+		if podScheduled(pod) && util.PodWantsNetwork(pod) && now.After(podTimer) {
```

Comment:
We never add hostnetwork pods to `retryPods` (and it's not possible to change `pod.Spec.HostNetwork` after creation time) so you shouldn't need to check `PodWantsNetwork()` here.

Reply:
ok will remove it